agent/src/main/scala/trackedfuture/agent/MethodAdapter.scala

```diff
@@ -2,77 +2,85 @@ package trackedfuture.agent
 
 import org.objectweb.asm._
 
-class TFMethodAdapter(up: MethodVisitor) extends MethodVisitor(Opcodes.ASM5, up) {
```

Comment:
If we rename TFMethodAdapter => MethodAdapter, why not rename TFClassAdapter => ClassAdapter ?
//of course, i can do this, just comment